vignettes/concepts_conventions.Rmd

```diff
@@ -23,7 +23,8 @@ library(admiraldev)
 This vignette aims to discuss some of the common programming concepts and conventions that have been adopted within the
 `{admiral}` family of packages. It is intended to be a user-facing version of the [Programming Strategy](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html) vignette, but users can also read the
 latter after becoming more familiar with the package to expand further on any topics of interest. For some of common
-`{admiral}` FAQ, visit the corresponding [FAQ page](faq.html).
```

Comment:
That's strange. We have similar links in other vignettes. I wonder why it is an issue here.

Reply:
yeah - it is weird.  maybe cause the concept vignette is actually not part of our website yet??  but I think there are other links...